packages/esm-patient-medications-app/src/api/api.ts

```diff
@@ -54,6 +53,51 @@ export function usePatientOrders(patientUuid: string, status: 'ACTIVE' | 'any')
   };
 }
 
+/**
+ * Hook to get active patient orders.
+ *
+ * @param patientUuid The UUID of the patient whose active orders should be fetched.
+ */
+export function useActivePatientOrders(patientUuid: string) {
+  const { data: allOrders, error, isLoading, isValidating, mutate } = usePatientOrders(patientUuid);
+
+  const activeOrders = useMemo(
+    () => (allOrders ? allOrders.filter((order) => !order.autoExpireDate && !order.dateStopped) : null),
+    [allOrders],
+  );
```

Comment:
This filtering is already done on the backend if `excludeCanceledAndExpired` is set to `true`, so it's redundant here.